sevntu-checks/src/main/resources/com/github/sevntu/checkstyle/checks/coding/messages.properties

```diff
@@ -59,3 +59,28 @@ useless.super.ctor.call.without.args= Redundant super constructor call could be
 return.null.Boolean=Method declares to return Boolean and returns null.
 return.boolean.ternary=Returning explicit boolean from ternary operator.
 whitespace.before.array.initializer=Array initializer should have whitespace before.
+jsr305.illegal.class.level.annotation=@CheckForNull, @Nullable, @Nonnull and @CheckReturnValue are not allowed on class level. Use @ParametersAreNonnullByDefault, @ParametersAreNullableByDefault and @ReturnValuesAreNonnullByDefault.
+jsr305.contradicting.class.level.annotations=@ParametersAreNullableByDefault and @ParametersAreNonnullByDefault are not allowed together!
+jsr305.param.definitions.with.check.annotation=Parameter definitions don't need checking, use @Nullable or @Nonnull.
+jsr305.param.definition.with.override.annotation=@Override is not allowed on parameter definition!
```

Comment:
please remove `!` at the end,  .... from all messages.

Reply:
Done with the next commit (I am still aggregating changes).